### PR TITLE
Add more methods for track.

### PR DESCRIPTION
**Pull request by cloudwebrtc**
Support mute/unmute for local track and set enable/disable for remote track.

### CODE REVIEW

File: livekit-ffi/src/server/requests.rs [hunk 1]
```diff
@@ -231,6 +231,70 @@ fn on_create_audio_track(
...
+        }
+        Track::RemoteVideo(track) => {
+            if request.enabled {
+                track.enable();
+            } else {
+                track.disable();
+            }
+            enabled = track.is_enabled();
+        }
+        _ => return Err(FfiError::InvalidRequest("track is not a local track".into())),
```
Comment (nbsp) [inline livekit-ffi/src/server/requests.rs]: s/local/remote/

File: livekit-ffi/src/server/requests.rs [hunk 2]
```diff
@@ -231,6 +231,70 @@ fn on_create_audio_track(
...
+        }
+        Track::LocalVideo(track) => {
+            if request.mute {
+                track.mute();
+            } else {
+                track.unmute();
+            }
+            muted = track.is_muted();
+        }
+        _ => return Err(FfiError::InvalidRequest("track is not a local track".into())),
```
Comment (nbsp) [inline livekit-ffi/src/server/requests.rs:258]: i think these errors should maybe show `track_sid` for the user

**Review by nbsp — APPROVED** [review]
looks good, just a few nitpicks below

+ run cargo fmt

**Review by theomonnom — APPROVED** [review]

File: livekit-ffi/src/server/requests.rs (hunk 2 above)
Comment (theomonnom) [reply livekit-ffi/src/server/requests.rs:258]: not necessary, it will just means our implementation is incorrect

**Pull request merged**